src/PublicApiAnalyzers/Core/CodeFixes/DeclarePublicApiFix.cs

```diff
@@ -269,7 +270,7 @@ public override async Task<CodeAction> GetFixAsync(FixAllContext fixAllContext)
                         {
                             ImmutableArray<Diagnostic> diagnostics = await fixAllContext.GetDocumentDiagnosticsAsync(fixAllContext.Document).ConfigureAwait(false);
                             diagnosticsToFix.Add(new KeyValuePair<Project, ImmutableArray<Diagnostic>>(fixAllContext.Project, diagnostics));
-                            title = string.Format(titleFormat, "document", fixAllContext.Document.Name);
+                            title = string.Format(CultureInfo.InvariantCulture, titleFormat, "document", fixAllContext.Document.Name);
```

Comment:
📝 A bug should be filed to fix this method. It's similar to dotnet/roslyn#39411.

Reply:
https://github.com/dotnet/roslyn-analyzers/issues/2976